res/openvpn-gui-res-it.rc

```diff
@@ -172,50 +187,50 @@ END
 /* Advanced Dialog */
 ID_DLG_ADVANCED DIALOGEX 6, 18, 252, ADVANCED_DIALOG_HEIGHT
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
-CAPTION "Avanzate"
+CAPTION "Opzioni avanzate"
 FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
-    GROUPBOX "File di configurazione", 201, 6, 12, 235, 45
+    GROUPBOX "File configurazione", 201, 6, 12, 235, 45
     LTEXT "&Cartella:", ID_TXT_FOLDER, 17, 25, 32, 10
     EDITTEXT ID_EDT_CONFIG_DIR, 53, 23, 150, 12, ES_AUTOHSCROLL
     LTEXT "&Estensione:", ID_TXT_EXTENSION, 17, 40, 52, 10
     EDITTEXT ID_EDT_CONFIG_EXT, 53, 38, 25, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_CONFIG_DIR, 208, 23, 25, 12
 
-    GROUPBOX "File di log", 202, 6, 62, 235, 30
+    GROUPBOX "File registro", 202, 6, 62, 235, 30
     LTEXT "C&artella:", ID_TXT_FOLDER1, 17, 74, 32, 10
     EDITTEXT ID_EDT_LOG_DIR, 53, 72, 150, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_LOG_DIR, 208, 72, 25, 12
 
     GROUPBOX "Timeout script", 203, 6, 97, 235, 60
-    LTEXT "Timeout script di &preconnessione:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 150, 10
+    LTEXT "Timeout script &preconnessione:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 150, 10
     EDITTEXT ID_EDT_PRECONNECT_TIMEOUT, 183, 108, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
-    LTEXT "Timeout script di c&onnessione:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 150, 10
+    LTEXT "Timeout script c&onnessione:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 150, 10
     EDITTEXT ID_EDT_CONNECT_TIMEOUT, 183, 123, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
-    LTEXT "Timeout script di &disconnessione:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 150, 10
+    LTEXT "Timeout script &disconnessione:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 150, 10
     EDITTEXT ID_EDT_DISCONNECT_TIMEOUT, 183, 138, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
 
-    GROUPBOX "Management interface", 204, 6, 163, 235, 30
-    LTEXT "Port offset:", 205, 17, 175, 75, 10
+    GROUPBOX "Gestione interfaccia", 204, 6, 163, 235, 30
+    LTEXT "Offset porta:", 205, 17, 175, 75, 10
     EDITTEXT ID_EDT_MGMT_PORT, 103, 173, 30, 12, ES_AUTOHSCROLL
 
-    GROUPBOX "Config menu view", 206, 6, 198, 235, 30
-    AUTORADIOBUTTON "&Auto", ID_RB_BALLOON0, 28, 210, 50, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "&Flat", ID_RB_BALLOON1, 88, 210, 50, 10
-    AUTORADIOBUTTON "&Nested", ID_RB_BALLOON2, 148, 210, 50, 10
+    GROUPBOX "Tipo menu menu configurazione", 206, 6, 198, 235, 30
+    AUTORADIOBUTTON "&Automatico", ID_RB_BALLOON0, 28, 210, 50, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "&Piatto", ID_RB_BALLOON1, 88, 210, 50, 10
+    AUTORADIOBUTTON "&Nidificato", ID_RB_BALLOON2, 148, 210, 50, 10
 
-    GROUPBOX "Echo message display", 207, 6, 233, 235, 45
-    AUTORADIOBUTTON "A&uto", ID_RB_BALLOON3, 28, 245, 50, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "Ne&ver", ID_RB_BALLOON4, 88, 245, 50, 10
-    LTEXT "Repeated messages are muted for: ", 208, 17, 260, 125, 10
+    GROUPBOX Visualizzazione emssaggio echo, 207, 6, 233, 235, 45
```

Comment:
Missing quotes around the text.

Reply:
I will fix as
GROUPBOX "Visualizzazione messaggio echo"

Reply:
Fixed